Diz.Core/util/ObservableDictionaryAdaptor.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using ExtendedXmlSerializer;
+using ExtendedXmlSerializer.Configuration;
+using IX.Observable;
+
```

Comment:
this entire file can go completely away when I figure out the right way to deal with ObservableDictionary and XML serialization.   tracking this here: https://github.com/ExtendedXmlSerializer/home/issues/463